analyzer/tools/compilation-database-transformer/LICENSE.txt

```diff
@@ -0,0 +1,39 @@
+University of Illinois/NCSA Open Source License
```

Comment:
Please update this file because the license is changed. For more information see: #2697